packages/lib-classifier/src/helpers/createLocationCounts/createLocationCounts.js

```diff
@@ -1,14 +1,26 @@
+import validateMimeType from '../validateMimeType'
+
 function createLocationCounts ({ locations }) {
-  const types = locations
+  const mimeTypes = locations
     .map(location => Object.keys(location)[0])
 
-  const images = types
-    .filter(type => type.startsWith('image'))
-    .length
+  try {
+    const images = mimeTypes
+      .filter(mimeType => mimeType.startsWith('image') && validateMimeType(mimeType))
+      .length
+
+    const json = mimeTypes
```

Comment:
You often see `.js` and `.json` sent as `text/plain` on the web. I think PH-TESS subjects use `text/plain` for JSON data.

Reply:
I think TESS using that was due to a bug in the python client.

Reply:
I didn't touch the getter that sets the viewer to use the LCV subject viewer to use the validation yet, so this is setup for VSLCV to validate properly and we can speak to the TESS team to make sure future sectors get uploaded with the correct mime type.

Reply:
Sounds good. You may find that the API client uploads `.json` as text, depending on where it's run and what type that machine/OS assigns to JSON files, but we can deal with that when it happens.